worker/modelcache/worker_test.go

```diff
@@ -82,6 +88,27 @@ func (s *WorkerSuite) TestConfigMissingCleanup(c *gc.C) {
 	c.Check(err, gc.ErrorMatches, "missing cleanup func not valid")
 }
 
+func (s *WorkerSuite) TestConfigNegativeMinRestartDelay(c *gc.C) {
```

Comment:
Would we want to disallow a restart delay of 0 as well? Since that would mean we never back off?